stable/kong/templates/cleanup-crds.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.ingressController.cleanUpCRDs }}
+apiVersion: batch/v1
```

Comment:
I've serious concerns of doing this as it might break existing deployments and managing definitions themselves in this way is way more riskier.
Let's remove this for now.

The `KongPlugin` created by Helm should be managed by Helm, meaning, Helm will delete it as part of this release.

Reply:
Agree